docker-compose.yml

```diff
@@ -2,11 +2,17 @@ version: '3.8'
 services:
   api:
     image: italia/developers-italia-api
+    ports:
+      - "3000:3000"
+    depends_on:
+      - db
+    networks:
+      - default
     build:
       context: .
       dockerfile: Dockerfile
-    environment:
-      DATABASE_DSN: host=localhost user=postgres password=postgres dbname=postgres port=5432
+    env_file:
+      - .env
```

Comment:
I'm not convinced about this. This is a docker-compose for the dev environment and we are not really expecting to change the database. Besides, we shoudn't commit an `.env` file.

Reply:
Ok, that's right. We can revert to standard envs